src/io/flutter/utils/StopWatch.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.utils;
+
+public class StopWatch {
```

Comment:
Aside: I can't remember but is this essentially a pared down version of the apache one?  (I'm guessing we want to avoid an additional dep?)